app/src/main/java/org/ieeeguc/ieeeguc/fragments/user/UserShow.java

```diff
@@ -1,23 +1,94 @@
 package org.ieeeguc.ieeeguc.fragments.user;
-
-
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-
+import android.widget.TextView;
+import org.ieeeguc.ieeeguc.HTTPResponse;
 import org.ieeeguc.ieeeguc.R;
+import org.ieeeguc.ieeeguc.controllers.MainActivity;
+import org.ieeeguc.ieeeguc.models.User;
+import org.json.JSONObject;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 
 /**
  * A simple {@link Fragment} subclass.
  */
 public class UserShow extends Fragment {
+    User u;
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_user_show, container, false);
+        final View view = inflater.inflate(R.layout.fragment_user_show, container, false);
+        MainActivity.loggedInUser.getUser(MainActivity.token,MainActivity.loggedInUser.getId(), new HTTPResponse() {

```

Comment:
Instead of calling this method using as a logged in user ID use the id that you should pass from the main activity